api/api.php

```diff
@@ -476,12 +476,14 @@
     // =============================================================================
     // USERS
     // =============================================================================
-    $app->map('/users/?', '\Directus\API\Routes\A1\Users:all')
-        ->via('GET', 'POST', 'PUT');
-    $app->map('/users/:id/?', '\Directus\API\Routes\A1\Users:get')
-        ->conditions(['id' => '[0-9]+'])
-        ->via('DELETE', 'GET', 'PUT', 'PATCH');
+    $app->get('/users/?', '\Directus\API\Routes\A1\Users:all');
+    $app->get('/users/:id/?', '\Directus\API\Routes\A1\Users:get')
+        ->conditions(['id' => '[0-9]+']);
     $app->post('/users/invite/?', '\Directus\API\Routes\A1\Users:invite');
+    $app->map('/users/:id/?', '\Directus\API\Routes\A1\Users:update')
+        ->conditions(['id' => '[0-9]+'])
+        ->via('DELETE', 'PUT', 'PATCH');
+    $app->post('/users/', '\Directus\API\Routes\A1\Users:update');
```

Comment:
Please add the question mark, (`/users/?`) so it works with or without the trailing slash.